DTS-STN/Alpha-Site
# Update splash page

# Description

[Splash Page EN/FR](https://trello.com/c/585Am4Th/369-369-splash-page-en-fr)

I updated the splash page to the match most recent Figma designs.
![image](https://user-images.githubusercontent.com/72703030/125806572-7f7851f3-9219-41fe-8e9c-63a482fd1ba7.png)
![image](https://user-images.githubusercontent.com/72703030/125806739-a00dacf7-2f60-4abb-b4b6-930d44a679fa.png)

## Acceptance Criteria

The splash page should look like the Figma design.

## Test Instructions

1. Run the site and go to the splash page
2. Compare it to the Figma design
3. Also check for different screen sizes

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
